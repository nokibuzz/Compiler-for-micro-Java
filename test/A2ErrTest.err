ERROR 01:45:11,035 - 
Sintaksna greska na liniji 10
ERROR 01:45:11,035 - Izvrsen oporavak na liniji 8
ERROR 01:45:11,050 - Parsiranje NIJE uspesno zavrseno!
