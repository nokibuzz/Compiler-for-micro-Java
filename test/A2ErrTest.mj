program testA2e

	int a, b;
	int err1;
{
	void main()	{
		
		a = 6;
		b = 7;
		err1 >= 8;
	}
}