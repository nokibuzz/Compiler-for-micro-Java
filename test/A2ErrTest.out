INFO  01:45:11,003 - Compiling source file: C:\Users\Petar\Desktop\np150264d - pp\pp1lab.templateAST\workspace\MJCompiler\test\testA2e.mj
INFO  01:45:11,035 - #2 program
INFO  01:45:11,035 - #43 testA2e
INFO  01:45:11,035 - #43 int
INFO  01:45:11,035 - #43 a
INFO  01:45:11,035 - #4 ,
INFO  01:45:11,035 - #43 b
INFO  01:45:11,035 - #3 ;
INFO  01:45:11,035 - #43 int
INFO  01:45:11,035 - #43 err1
INFO  01:45:11,035 - #3 ;
INFO  01:45:11,035 - #7 {
INFO  01:45:11,035 - #13 void
INFO  01:45:11,035 - #43 main
INFO  01:45:11,035 - #9 (
INFO  01:45:11,035 - #10 )
INFO  01:45:11,035 - #7 {
INFO  01:45:11,035 - #43 a
INFO  01:45:11,035 - #12 =
INFO  01:45:11,035 - #42 6
INFO  01:45:11,035 - #3 ;
INFO  01:45:11,035 - #43 b
INFO  01:45:11,035 - #12 =
INFO  01:45:11,035 - #42 7
INFO  01:45:11,035 - #3 ;
INFO  01:45:11,035 - #43 err1
INFO  01:45:11,035 - #35 >=
ERROR 01:45:11,035 - 
Sintaksna greska na liniji 10
INFO  01:45:11,035 - #42 8
INFO  01:45:11,035 - #3 ;
INFO  01:45:11,035 - #8 }
INFO  01:45:11,035 - #8 }
ERROR 01:45:11,035 - Izvrsen oporavak na liniji 8
INFO  01:45:11,050 - Deklarisana promenljiva a na liniji 3
INFO  01:45:11,050 - Deklarisana promenljiva b na liniji 3
INFO  01:45:11,050 - Povratna vrednost deklarisane promenljive je int na liniji 3
INFO  01:45:11,050 - Deklarisana promenljiva err1 na liniji 4
INFO  01:45:11,050 - Povratna vrednost deklarisane promenljive je int na liniji 4
INFO  01:45:11,050 - Obradjuje se void funkcija main na liniji 6
INFO  01:45:11,050 - Nadjena main metoda
INFO  01:45:11,050 - Print calls = 0
ERROR 01:45:11,050 - Parsiranje NIJE uspesno zavrseno!
